constellations/orion/src/domains/family.go

```diff
@@ -0,0 +1,33 @@
+package domains
+
+import (
+	"database/sql"
+	"errors"
+	"regexp"
+	"time"
+)
+
+var TABLE_FAMILY = "family"
+
+//ID,password and primary contact email
+//possibly add family name?
+type Family struct {
+	Id         uint         `json:"id"`
+	CreatedAt  time.Time    `json:"-" db:"created_at"`
+	UpdatedAt  time.Time    `json:"-" db:"updated_at"`
+	DeletedAt  sql.NullTime `json:"-" db:"deleted_at"`
+	PrimaryEmail string     `json:"primaryEmail db:"primary_email"` 
+	Password   string       `json:"password" db:"password"`
+}
+
+// Class Methods
+
+func (family *Family) Validate() error {
+	// Retrieves the inputted values
+	password := family.Password
+
+	if password == "" || len(password) > 32 {
```

Comment:
hmm, let's just let the DB take care of password length. So if the column only wants to keep 256 bytes of characters, if the password size goes over, the db will signal an error.

Reply:
so remove the `len(password) >` part